packages/esl/src/esl-carousel/plugin/autoplay/esl-carousel.autoplay.mixin.ts

```diff
@@ -112,6 +118,12 @@ export class ESLCarouselAutoplayMixin extends ESLCarouselPlugin<ESLCarouselAutop
     return this.$interactionScope.some(($el) => $el.matches('*:hover'));
   }
 
+  /** True if active slide contains any blocking items */
+  public get hasActiveBlockingItems(): boolean {
+    const {blockerSelector} = this.config;
+    return !!blockerSelector && this.$interactionScope.some(($el) => ESLTraversingQuery.first(blockerSelector, $el) !== null);
```

Comment:
Actually if we use traversing query, I'd vote for  
```suggestion
    return !!blockerSelector && !!ESLTraversingQuery.first(blockerSelector, this.$host);
```
Which is allows to do everything for user

Also if we pass just two arguents we need to fix the default query to
`::find(esl-share[active], esl-note[active])`
(othervice we need a third argument)